src/lib/components/TextAppearenceSelector.svelte

```diff
@@ -0,0 +1,35 @@
+<!--
+@component
+The navbar component.
+-->
+<script>
+    import Dropdown from './Dropdown.svelte';
+    import { TextAppearanceIcon } from '$lib/icons';
+    import { globalConfig } from '$lib/data/stores';
+
+    /** Checks if normal is enabled*/
+    $: countThemesNormal = $globalConfig.themes.find((x) => x.name === 'Normal').enabled;
+
+    /** Checks if sepia is enabled*/
+    $: countThemesSepia = $globalConfig.themes.find((x) => x.name === 'Sepia').enabled;
+
+    /** Checks if dark is enabled*/
+    $: countThemesDark = $globalConfig.themes.find((x) => x.name === 'Dark').enabled;
+
+    let showTextAppearence =
```

Comment:
This logic isn't quite right.  It should be:
* is there at least 2 colors or font-size-slider or line-height-slider